build/build-buildimage-bases.sh

```diff
@@ -7,59 +7,71 @@
 # This script builds some base images that are needed for the build image:
 # - Python binaries
 # - PHP binaries
+# - Yarn package cache
 #
 
 set -e
 
 source __variables.sh
 
+IMAGE_DIR_TO_BUILD=$1
 IMAGE_TAG="${BUILD_NUMBER:-latest}"
+BUILD_IMAGE_PREFIX="$__REPO_DIR/images/build"
 
 # Clean artifacts
 mkdir -p `dirname $BUILD_IMAGE_BASES_ARTIFACTS_FILE`
```

Comment:
Based on my other comment, we can make this file name dynamic which uses the `IMAGE_DIR_TO_BUILD` to construct it